app/controllers/ips_controller.rb

```diff
@@ -41,7 +48,7 @@ def publish_for_performance_platform
         bucket: ENV.fetch('S3_PUBLISHED_LOCATIONS_IPS_BUCKET'),
         key: ENV.fetch('S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY')
       ),
-      source_gateway: Gateways::Ips.new
+      source_gateway: Gateways::IpsGateway.new
```

Comment:
Had to rename this because of naming conflicts on `Ips` constant.

Reply:
(It's possible to fix this, but I think it's more faff than just doing this rename dodge)

Reply:
What is the Ips constant?  Aren't namespaces solving this already?